strawberry/experimental/pydantic/_compat.py

```diff
@@ -17,6 +17,9 @@
 IS_PYDANTIC_V2: bool = PYDANTIC_VERSION.startswith("2.")
 IS_PYDANTIC_V1: bool = not IS_PYDANTIC_V2
 
+if IS_PYDANTIC_V2 and TYPE_CHECKING:
+    import pydantic.v1
```

Comment:
I think we can remove this 😊

Reply:
interesting, can you remove it anyway? I'll double check what's wrong

Reply:
done!